File: java/src/test/java/kms/notice/TestGetEventNoticeList.java
package kms.notice;

import dev.spiralmoon.maplestory.api.kms.MapleStoryApi;
import dev.spiralmoon.maplestory.api.kms.dto.notice.EventNoticeListDTO;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

@DisplayName("getEventNoticeList")
public class TestGetEventNoticeList {
    private static final String apiKey = System.getProperty("API_KEY_KMS"); // Your API Key
    private static final MapleStoryApi api = new MapleStoryApi(apiKey);

    @Test
    @DisplayName("success: getEventNoticeList")
    void getEventNoticeList() {
        EventNoticeListDTO response = api.getEventNoticeList().join();
        System.out.println(response.toString());
    }
}

File: csharp/MapleStory.Test/KMS/Notice/TestGetEventNoticeList.cs
using NUnit.Framework;
using MapleStory.OpenAPI.KMS;

namespace MapleStory.Test.KMS.Notice
{
    [TestFixture]
    public class TestGetEventNoticeList
    {
        private static readonly string apiKey = EnvConfig.Get("API_KEY_KMS"); // Your API Key
        private static readonly MapleStoryAPI api = new MapleStoryAPI(apiKey);

        [Test, Description("success: GetEventNoticeList")]
        public async Task GetEventNoticeList()
        {
            var response = await api.GetEventNoticeList();
            Console.WriteLine(response.ToJson());
        }
    }
}
